boards.txt

```diff
@@ -39636,3 +39636,43 @@ circuitart_zero_s3.menu.EraseFlash.all=Enabled
 circuitart_zero_s3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+codecell.name=CodeCell
+codecell.vid.0=0x303a
+codecell.pid.0=0x1002
```

Comment:
You are missing those 2 lines, as IDE1 and IDE2 require different lines for VID/PID.
```suggestion
codecell.vid.0=0x303a
codecell.pid.0=0x1002
codecell.upload_port.0.vid=0x303a
codecell.upload_port.0.pid=0x1002
```